_layouts/page.html

```diff
@@ -4,16 +4,16 @@
 
 <div class="wrapper">
     <div class="page-content container-fluid">
-		<article class="post">
+		<div class="panel panel-default">
 
-		  <header class="post-header">
-		    <h1 class="post-title">{{ page.title }}</h1>
-		  </header>
+		  <div class="panel-header">
+		    <h1 class="page2-title">{{ page.title }}</h1>
```

Comment:
This is why we code review :)
accidental careless holdover from some earlier experimenting, I'll remove that part~!